src/DDTrace/Integrations/Predis.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace DDTrace\Integrations;
+
+use DDTrace\Tags;
+use DDTrace\Types;
+use OpenTracing\GlobalTracer;
+use Predis\Client;
+
+const VALUE_PLACEHOLDER = "?";
+const VALUE_MAX_LEN = 100;
+const VALUE_TOO_LONG_MARK = "...";
+const CMD_MAX_LEN = 1000;
+
+class Predis
+{
+    /**
+     * Static method to add instrumentation to the Predis library
+     */
+    public static function load()
+    {
+        if (!extension_loaded('ddtrace')) {
+            trigger_error('ddtrace extension required to load Predis integration.', E_USER_WARNING);
+            return;
+        }
+
+        // public mixed Predis\Client::executeCommand(CommandInterface $command)
+        dd_trace(Client::class, 'executeCommand', function ($command) {
+            $query = Predis::formatCommandArgs($command);
+            $scope = GlobalTracer::get()->startActiveSpan('Predis.executeCommand');
+            $span = $scope->getSpan();
+            $span->setTag(Tags\SPAN_TYPE, Types\REDIS);
+            $span->setTag(Tags\SERVICE_NAME, 'redis');
+            $span->setTag('redis.raw_command', $query);
```

Comment:
It might be worthwhile to sanitize this tag to avoid cardinality and data leak problems

Reply:
Are these not sanitized in the agent?